src/main/java/net/sf/jabref/logic/search/rules/RegexBasedSearchRule.java

```diff
@@ -51,9 +52,13 @@ public boolean applyRule(String query, BibEntry bibEntry) {
 
         for (String field : bibEntry.getFieldNames()) {
             if (bibEntry.hasField(field)) {
-                String fieldContent = RegexBasedSearchRule.REMOVE_LATEX_COMMANDS
-                        .format(bibEntry.getFieldOptional(field).get());
-                String fieldContentNoBrackets = RegexBasedSearchRule.REMOVE_LATEX_COMMANDS.format(fieldContent);
+                Optional<String> fieldOptional = bibEntry.getFieldOptional(field);
```

Comment:
Or you can simply use:
```
bibEntry.getFieldOptional.ifPresent(field->  {
do stuff 
} );
```
Then the else would be not necessary, because it would automatically continue with the next entry